src/app/shared/mydspace-actions/workspaceitem/workspaceitem-actions.component.ts

```diff
@@ -77,6 +100,25 @@ export class WorkspaceitemActionsComponent extends MyDSpaceActionsComponent<Work
     );
   }
 
+
+  ngOnInit(): void {
+    const activeEPerson$ = this.authService.getAuthenticatedUserFromStore();
+
+    this.isAdmin$ = activeEPerson$.pipe(
+      switchMap((user: EPerson) => this.authorizationService.isAuthorized(FeatureID.AdministratorOf, user.uuid)));
```

Comment:
I think this `isAuthorized()` call is incorrect, as I'm seeing an error in my browser's DevTools console.  The backend returns `400 Bad Request` on this line.

I believe it should simply be:
```
this.isAdmin$ = this.authorizationService.isAuthorized(FeatureID.AdministratorOf);
```
(See for example `item-page.component.ts` which has this same code).